plugins/base/src/main/resources/dokka/scripts/platformContentHandler.js

```diff
@@ -19,6 +19,11 @@ function initTabs(){
             showCorrespondingTabBody(element)
             element.addEventListener('click', (event) => toggleSections(event))
         })
+    let cached = localStorage.getItem("active-tab")
```

Comment:
Hmm that means that there is only one entry for all pages, so when i change tab on 1 page it will be the same on the others.

I don't think it should work like that. You can add a location to the end of the key to make it page-specific.

Same thing goes for the sourceset filters, but imho persisting them between pages i nice.

Reply:
Also you can just do `if(cached)` and why do you need to `JSON.parse` it when it is a string?

Reply:
I have spoken with @Kordyjan about scope of cache and if I understood correctly, we decided to make filters cache to work between pages.

About comparison - I never know how to check nulls and undefined's correctly.

About JSON.parse - It's true, I didn't realized that it is just string and my default behavior is to call JSON.parse when retrieving data from localStorage.

Reply:
Fixed null-checks